### DISCUSSION TITLE
Development version not working

### DISCUSSION THREAD

**Discussion by Bodo0210** [flybywiresim/aircraft#4324] [Help]
Hello,
I have aproblem with the experimental version. After the last update of it, I only get the Standard Airbus from Microsoft. The same problem with the development version. The stable version seems to be running.
Any Ideas?
Greetings Bodo

**Comment by jachu1978** [comment]
https://github.com/flybywiresim/a32nx/discussions/4246

**Reply by Bodo0210** [reply]
Thanks for the fast help, everything is working fine now.
